client/src/pages/Home.jsx

```diff
@@ -57,14 +58,6 @@ const Home = () => (
           linkTo="/election-day"
         />
       </Col>
-      <Col md={6} lg={4} sm={6}>
-        <Card
-          title="Find Out Who Your Candidates Are"
-          body="Confused about who you are voting for? To find your sample ballot click below"
-          btnText="Find your candidates &amp; sample ballot"
-          linkTo="/candidates"
-        />
-      </Col>
```

Comment:
In keeping with your other commented out code, you might want to comment out this too vs deleting it.

Alternatively, you can just put a link to this GH issue / PR as a comment about Candidate info being removed instead of leaving commented out code elsewhere, so we can refer to the issue / PR later in the future.

Since we are using git for source control, it is preferred to delete code vs commenting it out.  I think the happy medium is adding a useful comment to point us in the right direction should we want to do a little research and add it back later.

Reply:
Good call and thanks for educating. I'm still pretty new to a lot of this. I'll make it consistent so that both edits are done with a comment linking to the PR/issue and delete the code instead of commenting out.